test/test-helpers/fake-data.js

```diff
@@ -308,18 +308,29 @@ export const fakeComment = async commentData => {
 /**
  * Creates a fake comment reaction. All params are optionals.
  */
-export const fakeCommentReaction = async (reactionData = {}) => {
+export const fakeEmojiReaction = async (isComment, reactionData = {}) => {
```

Comment:
Let's keep a similar API to the other `fake*` helpers above.

```suggestion
export const fakeEmojiReaction = async (reactionData = {}, opts) => {
```

Reply:
@Betree : Did you mean passing an objection `opt` as second argument? I've done it that way, but let me know if you see any further issues. 😄